java/src/main/java/system/gameplay/engine/io/DataPrinter.java

```diff
@@ -23,5 +24,15 @@ public static void printAsOrderedList(List<?> list) {
         }
     }
 
+    /**
```

Comment:
For the JavaDoc, this precisely describes the one way we're using the method right now, but I meant that you should have the doc to describe the generic use of the method